implementations/rust/ockam/ockam_api/src/nodes/service/portals.rs

```diff
@@ -55,7 +55,7 @@ impl NodeManager {
 
         // Check that there is no entry in the registry with the same alias
         if self.registry.outlets.contains_key(&alias) {
-            let message = format!("A TCP outlet with alias '{alias}' already exists");
+            let message = format!("A service with alias '{alias}' already exists");
```

Comment:
The `service` concept is only relevant in the `ockam_app` context. In the `ockam_api`, an outlet is still an outlet, so we don't want to use the term `service` here.